source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -1010,7 +1010,7 @@ void AggressiveDCEPass::InitExtensions() {
       "SPV_NV_bindless_texture",
       "SPV_EXT_shader_atomic_float_add",
       "SPV_EXT_fragment_shader_interlock",
-      "SPV_NV_compute_shader_derivatives",
+      "SPV_KHR_compute_shader_derivatives",
```

Comment:
I think this should be additive. The capabilities and execution modes are aliases, but these strings are not. So in all these passes it makes sense to list both as handled.

Reply:
Ah ok, makese sense